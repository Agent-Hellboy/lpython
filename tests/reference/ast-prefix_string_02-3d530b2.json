{
    "basename": "ast-prefix_string_02-3d530b2",
    "cmd": "lpython --show-ast --no-color {infile} -o {outfile}",
    "infile": "tests/errors/prefix_string_02.py",
    "infile_hash": "5d0c279ea735e60d5243a4b33100832dc1564917d6ef83c9b32705f9",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "ast-prefix_string_02-3d530b2.stdout",
    "stdout_hash": "c80f1f341d99eb446dd3b339ee303493bd623e3696522c2923486002",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}