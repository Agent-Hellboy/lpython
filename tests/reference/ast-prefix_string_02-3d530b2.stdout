(Module
    [(ImportFrom
        lpython
        [(i32
        ())]
        0
    )
    (FunctionDef
        main
        ([]
        []
        []
        []
        []
        []
        [])
        [(Expr
            (Call
                (Name
                    print
                    Load
                )
                [(ConstantStr
                    "Hello World"
                    ()
                )]
                []
            )
        )]
        []
        ()
        ()
    )
    (Expr
        (Call
            (Name
                main
                Load
            )
            []
            []
        )
    )]
    []
)
